### PR TITLE
Feature/check quick hover bug

### PR DESCRIPTION
**Pull request by Eugen1992**
Rebased. Had delete branch on remote.
Build has passed ok.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,10 +49,13 @@
     "chai-spies": "^0.7.1",
     "coveralls": "^2.11.15",
     "enzyme": "^2.6.0",
-    "eslint": "^3.10.2",
+    "eslint": "3.10.2",
+    "eslint-config-airbnb": "13.0.0",
     "eslint-config-standard": "^6.2.1",
```
Comment (galk-in) [inline package.json]: Do we need this dependency? Maybe make clean up?

Comment (galk-in) [inline package.json]: eslint-config-standard!!!

**Comment by galk-in** [comment]
@Eugen1992 please check package.json dependecies.

File: package.json [hunk 2]
```diff
@@ -50,7 +50,9 @@
     "coveralls": "^2.11.15",
     "enzyme": "^2.6.0",
     "eslint": "^3.10.2",
-    "eslint-config-standard": "^6.2.1",
+    "eslint-config-airbnb": "13.0.0",
+    "eslint-plugin-import": "2.2.0",
+    "eslint-plugin-jsx-a11y": "2.2.3",
     "eslint-plugin-promise": "^3.4.0",
     "eslint-plugin-react": "^6.4.1",
     "eslint-plugin-standard": "^2.0.1",
```
Comment (galk-in) [inline package.json]: Why we need this dependency?

File: src/menu-item/menu-item.js [hunk 3]
```diff
@@ -1,80 +1,95 @@
...
+  handleMouseEnter(e) {
+    this.props.mouseOverHandler(e.currentTarget);
+  }
+  handleMouseLeave(e) {
+    const currentTarget = e.currentTarget;
+    const { mouseOutHandler } = this.props;
+    mouseOutHandler(currentTarget);
+  }
+  renderInnerList() {
+    let result;
```
Comment (galk-in) [inline src/menu-item/menu-item.js]: do we can use `return` without making this unused variable?

File: package.json (hunk 2 above)
Comment (Eugen1992) [inline package.json]: In order to run tests with airbnb config

File: src/menu-item/menu-item.js (hunk 3 above)
Comment (Eugen1992) [inline src/menu-item/menu-item.js]: We can. But default eslint settings do not allow it and I personally find it more readable.

File: package.json (hunk 2 above)
Comment (galk-in) [inline package.json]: I commented `eslint-plugin-standard` which not used by airbnb

Comment (Eugen1992) [inline package.json]: Tests fail without this dependency as well. I guess airbnb config relies on this one. I might check for more details.

**Comment by Eugen1992** [comment]
Cleaned commit history.

**Pull request merged**